src/scripts/shell/platform-choice/platform-choice.component.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
```

Comment:
@bhollis Bringing in the individual functions shouldn't be but a few bytes for each.  I'm not importing the entire library.